src/server/transaction.h

```diff
@@ -123,6 +123,7 @@ class Transaction {
   }
 
   void UnlockMulti();
+  void SetMultiUniqueShardCount();
```

Comment:
Please add a comment what this does and why it needs to be computed. All the transaction code is difficult and having one more place without comments makes it only more difficult to understand 😃

Reply:
So you check for a multi transaction on how many shards it has run overall. It seems like maybe there is some faster way to do this by maybe marking the shard runs in some kind of bitset when dispatching them.

I'm not an expert with transactions - thats a question to Roma. I guess its not important now - we can change it in the future.